vrc-get-gui/locales/zh_hant.json5

```
@@ -358,6 +367,38 @@
     "user packages:toast:package added": "套件添加成功",
     "user packages:toast:package removed": "套件移除成功。",
 
+    "packages:templates": "範本",
+    "templates:create template": "創建範本",
+    "templates:category": "類別",
+    "templates:id": "範本 ID",
+    "template:no id": "無 ID",
+    "templates:category:builtin": "內建",
+    "templates:tooltip:category:builtin": "內建範本由 ALCOM 開發者創建、隨 ALCOM 一同提供。",
+    "templates:category:alcom": "自訂",
+    "templates:tooltip:category:alcom": "自訂範本由你創建、可在 ALCOM 內編輯。",
+    "templates:category:vcc": "VCC",
+    "templates:tooltip:category:vcc": "VCC 範本由你手動為 VCC 創建。",
+    "templates:tooltip:remove template": "移除範本",
+    "templates:tooltip:remove builtin template": "不可移除內建範本。",
+    "templates:tooltip:remove vcc template": "你無法在此移除 VCC 範本，請至範本資料夾裡移除。",
+    "templates:menuitem:edit template": "編輯範本",
+    "templates:menuitem:export template": "匯出 範本",
+    "templates:tooltip:export template with unitypackage": "由於 Unity 套件（Unity packages）不具備可攜性，所以你無法匯出含有它們的模板。",
+    "templates:menuitem:open vcc template": "開啟範本資料夾",
+    "templates:dialog:edit template": "編輯範本",
+    "templates:dialog:create template": "創建範本",
+    "templates:dialog:general information": "基本資料",
+    "templates:dialog:base template": "基於",
+    "templates:dialog:unity version": "Unity 版本",
+    "templates:dialog:version": "版本",
+    "templates:dialog:unitypackages": "Unity 套件（.unitypackages）",
+    "templates:dialog:no unitypackages": "無 Unity 套件",
+    "templates:button:save template": "儲存範本",
+    "templates:button:import template": "匯入範本",
+    "templates:toast:imported n templates_one": "已匯入 {{count}} 個範本",
+    "templates:toast:imported n templates_other": "已匯入 {{count}} 個範本",
```

```suggestion
    "templates:toast:imported n templates": "已匯入 {{count}} 個範本",
```

Can be combined to one

---

do you want to keep using separated keys? @yelandiealma